libevmasm/Assembly.h

```diff
@@ -147,7 +148,8 @@ class Assembly
 
 	/// Create a JSON representation of the assembly.
 	Json::Value assemblyJSON(
-		std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>()
+		std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>(),
+		bool _includeSourceList = true
```

Comment:
Shouldn't this default to false to retain the old behaviour? Do we need a default value?

Reply:
Right now I set `includeSourceList` to `false`, only if a sub-assembly is exported. In `CompilerStack::assemblyJSON(..)` we want to have the source list included. If called with default value, the top-level assembly json will always include the source list, but it's sub-assemblies not.

Reply:
Oh, this should also be mentioned in the changelog.

Reply:
Thats true!